e2e/cases/svelte/package.json

```diff
@@ -4,5 +4,10 @@
   "version": "1.0.0",
   "dependencies": {
     "svelte": "^4.2.2"
+  },
+  "devDependencies": {
+    "less": "^4.2.0",
+    "sass": "^1.69.5",
```

Comment:
Rsbuild supports less and sass by default, are these dependencies read by `svelte-loader`?

Reply:
These are the peerDeps of svelte-preprocess 

https://github.com/sveltejs/svelte-preprocess/blob/main/package.json#L97

Reply:
get~